src/main/resources/db/h2/bootstrap_data.sql

```diff
@@ -63,3 +63,11 @@ VALUES
 (2, 'CASE', 1000, 'X02', 'How about being positive? :smile:', 0),
 
 (3, 'CASE', 1001, 'X03', 'Should we have done this instead? `x = 3`?', 1);
+
```

Comment:
Could you add a couple rows to make it easy to enroll the current user you're testing with in assignments. (This will help when you're testing the endpoints since you'll only be able to pull for the current user)

Add this code at the top of the file
```
-- Replace this id with your test user
SET @USER_ID = 'TEST USER ID'
```
And then enroll them
```
(6, 1, @USER_ID, false),
(7, 2, @USER_ID, true);
```

You can find your user id in keycloak for when you're testing endpoints later.